src/routes/index.svelte

```diff
@@ -52,9 +64,9 @@
 
     const updateSeries = history => {
         seriesList = []
-
         for (let value of history.values()) {
-            seriesList = [...seriesList, [...value.close]]
+            let prices = priceSelected === 'open' ? value.open : value.close
```

Comment:
```suggestion
            let prices = value[priceSelected]
```

Reply:
Lets also pass `priceSelected` as a parameter to `updateSeries` to keep the function using local variables.